db/migrate/20211109094557_create_participants.rb

```diff
@@ -0,0 +1,12 @@
+class CreateParticipants < ActiveRecord::Migration[6.1]
+  def change
+    create_table :participants do |t|
+      t.string :name
+      t.string :email
+      t.datetime :start_date
```

Comment:
but use it here?

Reply:
See above, converted the fikas attributes to datetime.